internal/controller/drplacementcontrol_controller.go

```diff
@@ -939,15 +941,21 @@ func getPlacementOrPlacementRule(
 		if err != nil {
 			return nil, err
 		}
+
+		placementType = "PlacementRule"
```

Comment:
I think the placementType assignments are swapped. This should be "Placement", since we're in the error block where PlacementRule wasn't found.